docs/_mitigations/mi-18_agent-authority-least-privilege-framework.md

```diff
@@ -0,0 +1,159 @@
+---
+sequence: 18
+title: Agent Authority Least Privilege Framework
+layout: mitigation
+doc-status: Draft
+type: PREV
+nist-sp-800-53r5_references:
+  - ac-6   # AC-6 Least Privilege
+  - ac-2   # AC-2 Account Management
+  - ac-3   # AC-3 Access Enforcement
+  - ac-5   # AC-5 Separation of Duties
+mitigates:
+  - ri-24  # Agent Action Authorization Bypass
+  - ri-18  # Model Overreach / Expanded Use
+related_mitigations:
+  - mi-12  # Role-Based Access Control for AI Data
+  - mi-3   # User/App/Model Firewalling/Filtering
+---
+
+## Purpose
```

Comment:
when an agent invokes an API or uses a tool, the caller role / identity needs to be included in the invocation - is that covered in this mitigation?

Reply:
I think it is implied, but not called out explicitly